appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/composable/Children.kt

```diff
@@ -28,12 +28,13 @@ import gestureModifier
 import kotlin.math.roundToInt
 
 @Composable
-inline fun <reified InteractionTarget : Any, ModelState : Any> ParentNode<InteractionTarget>.Children(
+fun <InteractionTarget : Any, ModelState : Any> Children(
+    parent: ParentNode<InteractionTarget>,
```

Comment:
for consistency, let's keep it an extension function on ParenNode?

Reply:
it was the extension on ParentNode that broke on JS - looks like there's a limited amount of scope nesting that the JS compilation system can take. That was a real headache to isolate! It causes a runtime crash without any stacktrace, so had to go literally line by line to infer what was going wrong.

There might be a way to do this in a way compatible with the platform but without there being docs or discussion about this it's hard to see how at this point. Though it is sad that it makes the api a little less neat.